index.html

```diff
@@ -57,6 +59,23 @@ <h2 class="hero__subtitle">A UX / UI designer, product designer, content creator
                 <div class="space"></div>
             </div>
         </div>
+
+        <div class="aboutme">
+            <h2 class="aboutme__title">Learn all about me in 60 seconds.</h2>
+            
+            <!-- <video class="aboutme__video" poster="./images/Aboutme/VideoPosterDesktop.svg" preload="auto" controls>
+	            <source src="http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4" type="video/mp4">
+            </video> -->
+
+            <div class="aboutme__video">
+                <picture>
+                    <source srcset="./images/Aboutme/VideoPosterDesktop.svg" media="(min-width: 1024px)">
+                    <source srcset="./images/Aboutme/VideoPosterTablet.svg" media="(min-width: 760px)">
+                    <img class="video__poster" src="./images/Aboutme/VideoPosterMobile.svg" alt="Video">
+                </picture>
+            </div>
+        </div>
+    </div>
```

Comment:
1. Would be more semantic to use `section`
2. Try to remove comments before pushing(except ones that explain the code)
3. I think that you should make a modified version of .container for this section and reuse the style, because there are multiple sections with the same width.